apps/cloudinary2/src/locations/Field/index.tsx

```diff
@@ -18,22 +18,26 @@ const Field = () => {
   const sdk = useSDK<FieldAppSDK<AppInstallationParameters>>();
   useAutoResizer({ absoluteElements: true });
 
-  const [assets = [], setAssets] = useFieldValue<CloudinaryAsset[]>(sdk.field.id, sdk.field.locale)
+  const [assets = [], setAssets] = useFieldValue<CloudinaryAsset[]>(sdk.field.id, sdk.field.locale);
+  const assetsIsNotNull = assets !== null;
 
   const [editingEnabled, setEditingEnabled] = useState(!sdk.field.getIsDisabled());
   useEffect(() => {
     sdk.field.onIsDisabledChanged((disabled) => setEditingEnabled(!disabled));
   }, [sdk.field]);
 
-  const canAddAssets = assets.length < sdk.parameters.installation.maxFiles && editingEnabled;
+  const canAddAssets = assetsIsNotNull && assets.length < sdk.parameters.installation.maxFiles && editingEnabled;
```

Comment:
Huh, I'm still not sure what it means for the field to be `null`. So if that's the case this line says you can't add more assets. Is that right?  🤔